stylesheets/components/nav.css

```diff
@@ -0,0 +1,49 @@
+.nav-container {
+  background-color: var(--primary-color);
+  color: white;
+  display: flex;
+  align-items: center;
```

Comment:
You can move this in a separate class as you're using it elsewhere too in this file

Reply:
made changes to use util